### PR TITLE
Authoring toolbar

### PR DESCRIPTION
**Pull request by Aca-jov**

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

File: scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -1206,18 +1211,43 @@
...
+                        }
+                    })
+                    .then(function (item) {
+                        if (item) {
+                            authoringWorkspace.close(false);
+                            notify.success(gettext('New take created.'));
+                            authoringWorkspace.edit(item);
+                        }
+                    }, function(err) {
+                        notify.error(gettext('Failed to send and continue.'));
```
Comment (mdhaman) [inline scripts/superdesk-authoring/authoring.js]: Change the message to `Failed to publish and continue`

File: scripts/superdesk-authoring/views/authoring-topbar.html [hunk 2]
```diff
@@ -15,74 +15,98 @@
...
-			 ng-if="highlight"
-			 translate>Export</button>
-
-        <button class="navbtn navbtn-hover strict"
-                ng-if="!item._locked && _editable"
-                title="{{ :: 'Minimize' | translate }}"
-                ng-click="minimize()">
+    <div class="subnav__stretch-bar">
+      <div class="subnav__button-stack subnav__button-stack--custom-buttons" ng-show="_editable">
+        <button class="btn btn-custom"
```
Comment (mdhaman) [inline scripts/superdesk-authoring/views/authoring-topbar.html]: You would need the some rules for the these buttons. `_editable` is not sufficient. 
- the user can either `correct` or `kill` the item after publishing.
- the user can only deschedule the item if the item is scheduled.
- user can only send content to a different stage if the content is not published.

Check the `itemActions` method authoring service.

**Comment by ioanpocol** [comment]
please rebase

File: scripts/superdesk-authoring/views/authoring-topbar.html [hunk 3]
```diff
@@ -15,74 +15,98 @@
...
+        translate>Edit and Kill</button>
+
+      <button class="btn btn--close" id="closeAuthoringBtn" ng-click="close()" translate title="{{ :: 'Close' | translate }}"><span class="btn__text">CLOSE</span><i class="icon-close-small btn__text--alt-icon"></i></button>
+
+      <button class="btn btn-info" type="submit"
+        ng-click="saveTopbar(item)"
+        ng-show="_editable"
+        ng-if = "itemActions.save && action === 'edit'"
+        ng-disabled="!save_enabled() || saveDisabled"
+        translate>SAVE</button>
```
Comment (ioanpocol) [inline scripts/superdesk-authoring/views/authoring-topbar.html]: Replace with Save

File: scripts/superdesk-authoring/views/authoring-topbar.html [hunk 4]
```diff
@@ -15,74 +15,98 @@
...
+        ng-show="action === 'view' && !item._locked && itemActions.correct"
+        ng-click="openAction('correct')"
+        translate>Edit and Correct</button>
+
+      <button class="btn btn-info" ng-if="item._type !== 'legal_archive'"
+        ng-show="action === 'view' && !item._locked && itemActions.kill"
+        ng-click="openAction('kill')"
+        translate>Edit and Kill</button>
+
+      <button class="btn btn--close" id="closeAuthoringBtn" ng-click="close()" translate title="{{ :: 'Close' | translate }}"><span class="btn__text">CLOSE</span><i class="icon-close-small btn__text--alt-icon"></i></button>
```
Comment (ioanpocol) [inline scripts/superdesk-authoring/views/authoring-topbar.html]: Replace CLOSE with Close

**Comment by vladnicoara** [comment]
LGTM

**Pull request merged**